projects/1.19/assets/reliquary-v1-3/reliquary/lang/zh_cn.json

```
@@ -0,0 +1,395 @@
+  "item.reliquary.ender_staff.tooltip.position": "当前绑定至X：%s Y：%s Z：%s",
+  "item.reliquary.ender_staff.tooltip.position.nowhere": "当前未绑定到任何位置。",
+  "item.reliquary.ender_staff.tooltip.position.out_of_range": "超出范围！",
+  "item.reliquary.fertile_potion": "肥沃药水",
+  "item.reliquary.fertile_potion.tooltip": "使植物生长。",
+  "item.reliquary.fortune_coin": "幸运硬币",
+  "item.reliquary.fortune_coin.tooltip": "吸取附近掉落物和经验",
+  "item.reliquary.fortune_coin.tooltip2": "Shift+右击激活。;按住右键开启超级吸取模式。",
+  "item.reliquary.glacial_staff": "冰河之杖",
+  "item.reliquary.glacial_staff.tooltip": "发射雪球，使敌人寒颤而无法行动，冻结你走过的地面。",
+  "item.reliquary.glacial_staff.tooltip2": "充能：%s",
+  "item.reliquary.glowing_water": "荧光之水",
+  "item.reliquary.glowing_water.tooltip": "能够伤害亡灵生物。",
+  "item.reliquary.glowing_bread": "荧光面包",
+  "item.reliquary.glowing_bread.tooltip": "完全回满饥饿值。",
+  "item.reliquary.grip_assembly": "握把配件",
+  "item.reliquary.grip_assembly.tooltip": "手枪部件",
+  "item.reliquary.hammer_assembly": "击锤配件",
+  "item.reliquary.hammer_assembly.tooltip": "手枪部件",
+  "item.reliquary.handgun": "猎人手枪",
+  "item.reliquary.handgun.tooltip": "为了伟大的正义而战。右击开火，长按装填。",
+  "item.reliquary.handgun.tooltip2": "%sx%s",
+  "item.reliquary.harvest_rod": "丰收之杖",
+  "item.reliquary.harvest_rod.tooltip": "右击催熟作物，可进行范围收割",
+  "item.reliquary.harvest_rod.tooltip2": "骨粉充能： %s",
+  "item.reliquary.harvest_rod.tooltip3": "%s：%s",
+  "item.reliquary.hero_medallion": "英雄之章",
+  "item.reliquary.hero_medallion.tooltip": "为你存储经验，右击取出经验。",
+  "item.reliquary.hero_medallion.tooltip2": "存有%s级经验+%s点经验值",
+  "item.reliquary.hero_medallion.tooltip.drain_levels": "右击取出%s级经验",
+  "item.reliquary.hero_medallion.tooltip.fill_stop_level": "等级为%s级时停止吸取",
+  "item.reliquary.hero_medallion.tooltip.scroll_to_change": "（Shift+滚轮改变等级数）",
+
+  "item.reliquary.holy_hand_grenade": "神圣手雷",
+  "item.reliquary.holy_hand_grenade.tooltip": "制造一次不会破坏方块，且不会对使用者造成伤害的猛烈爆炸。",
+
+  "item.reliquary.ice_magus_rod": "冰术士之杖",
+  "item.reliquary.ice_magus_rod.tooltip": "发射致伤的雪球。",
+  "item.reliquary.ice_magus_rod.tooltip2": "充能：%s",
+
+  "item.reliquary.infernal_chalice": "炼狱圣杯",
+  "item.reliquary.infernal_chalice.tooltip": "存储熔岩并保护你免受熔岩伤害",
+  "item.reliquary.infernal_chalice.tooltip2": "包含%smB熔岩",
+
+  "item.reliquary.infernal_claws": "炼狱之爪",
+  "item.reliquary.infernal_claws.tooltip": "以饥饿值为代价吸收火焰伤害。",
+  "item.reliquary.infernal_tear": "炼狱之泪",
+  "item.reliquary.infernal_tear.tooltip": "将物品烧炼成经验。",
+  "tooltip.reliquary.infernal_tear.absorb_unset": "右击移除物品关联",
+  "tooltip.reliquary.infernal_tear.tear_empty": "右击选取物品栏中最多的物品，使其可以被炼狱之泪烧炼。",
+  "item.reliquary.kraken_shell": "海妖壳甲",
+  "item.reliquary.kraken_shell.tooltip": "以饥饿度为代价吸收窒息伤害。",
+
+  "item.reliquary.lantern_of_paranoia": "诞妄提灯",
+  "item.reliquary.lantern_of_paranoia.tooltip": "自动在黑暗区域放置火把。Shift+右击激活。",
+  "item.reliquary.magicbane": "神魔克星",
+  "item.reliquary.magicbane.tooltip": "每条附魔都会增加其伤害。获得圣遗物中的掉落物的概率更高。",
+
+  "item.reliquary.magazines.empty_magazine": "空弹匣",
```

```suggestion
  "item.reliquary.magazines.empty_magazine": "空弹夹",
```
根据材质来说它更符合“弹夹”而非“弹匣”的定义